### ISSUE TITLE
Add prompty support

### ISSUE THREAD

**Issue by glharper** [Azure/azure-ai-cli#237]
Prompty is a defined file format that includes a chat prompt and metadata regarding the execution of that prompt. From Rob: "It has both the prompt (system and/or user prompt) as well as some "front-matter" that has meta data, things like author, model deployment name to be used, possibly variables that can be template instantiated into the prompt, with defaults, but they can be overriden, and possibly even "evaluation" data."

TL;DR: add *.prompty file support to AI CLI
